### DISCUSSION TITLE
how to know if my connection is under wss or if the keys key.pem and cert.pem are really running?

### DISCUSSION THREAD

**Discussion by skinkairewalker** [uNetworking/uWebSockets#1641] [General]
Hey guys, I'm trying to access my server example: EchoServer.cpp with wss:// but it doesn't connect, only with ws:// what am I doing wrong?

my vcpkg list : 
`D:\cppSolutions\vcpkg\vcpkg>vcpkg list
libuv:x64-windows                                 1.46.0              libuv is a multi-platform support library with a...
libuv:x86-windows                                 1.46.0              libuv is a multi-platform support library with a...
openssl:x64-windows                               3.1.1#1             OpenSSL is an open source project that provides ...
openssl:x86-windows                               3.1.1#1             OpenSSL is an open source project that provides ...
usockets:x64-windows                              0.8.6#1             Miniscule cross-platform eventing, networking & ...
usockets:x86-windows                              0.8.6#1             Miniscule cross-platform eventing, networking & ...
usockets[ssl]:x64-windows                                             Build usockets with openssl support
usockets[ssl]:x86-windows                                             Build usockets with openssl support
uwebsockets:x64-windows                           20.45.0             Simple, secure & standards compliant web I/O for...
uwebsockets:x86-windows                           20.45.0             Simple, secure & standards compliant web I/O for...
vcpkg-cmake-config:x64-windows                    2022-02-06#1
vcpkg-cmake-get-vars:x64-windows                  2023-03-02
vcpkg-cmake:x64-windows                           2023-05-04
zlib:x64-windows                                  1.2.13              A compression library
zlib:x86-windows                                  1.2.13              A compression library`

**Comment by opencnnet** [comment]
me too and listenSocket is null.

**Comment by uNetworkingAB** [comment]
EchoServer.cpp does not use SSL, so it makes no sense to assume it would accept wss. If you look at the example, it even tells you in comments how to make it run over TLS.
